aten/src/ATen/function_wrapper.py

```diff
@@ -130,6 +130,11 @@ def TypedDict(name, attrs, total=True):  # type: ignore
   .catchAllKernel<${return_type} (${formals_types})>(&TypeDefault::${api_name})
   .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
 """)
+DEFAULT_SCHEMA_REGISTRATION = CodeTemplate("""\
+.op(torch::RegisterOperators::options()
+  .schema("${schema_string}")
+  .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
```

Comment:
Not at all related to this PR but FYI I still really hate that you have to specify alias analysis at each operator registration site

Reply:
you soon don't have to anymore, Xintao is currently working on this.